tests/lib/rules/no-useless-escape.js

```diff
@@ -111,7 +111,7 @@ ruleTester.run("no-useless-escape", rule, {
         { code: "var foo = `\\'`;", parserOptions: {ecmaVersion: 6}, errors: [{ line: 1, column: 11, message: "Unnecessary escape character: \\'.", type: "TemplateElement"}] },
         { code: "var foo = `\\#`;", parserOptions: {ecmaVersion: 6}, errors: [{ line: 1, column: 11, message: "Unnecessary escape character: \\#.", type: "TemplateElement"}] },
         {
-            code: "var foo = '\\\`foo\\\`';",
+            code: "var foo = '\\\`foo\\\`';", // eslint-disable-line no-useless-escape
```

Comment:
Thanks, you're totally right! Think I thought the inner one was a single-quote too. Should use my slightly larger glasses next time :)